.github/workflows/client.yml

```diff
@@ -106,7 +119,7 @@ jobs:
           build-args: |
             REVISION=${{ github.sha }}
           # VERSION= ? TODO: Configure version, sample: 1.7.6
-          push: ${{ github.event_name == 'workflow_dispatch' }}
+          push: true
 
       - # Temp fix - move cache instead of copying (added below step and
```

Comment:
Can we move it to be the very last step (after `Notify CI about completion of the workflow`)?